queries.py

```diff
@@ -65,18 +112,20 @@ def print_result(result):
             .
         ?descriptor anno:start ?d_start
             ; anno:end ?d_end
+            ; anno:root/anno:pos ?pos
             .
+        FILTER (?pos NOT IN ("X", "PUNCT"))
         ?target anno:category "diseases"
             ; anno:start ?t_start
             ; anno:end ?t_end
             ; ^dc:relation ?rel
             .
+        ?rel rdf:label "malaria" .
```

Comment:
This is similar to the query used to populate the frequently used descriptor list in NIAM. There the label string interpolated into the template. I added the label here so I could debug strange descriptors appearing in the application.